### ISSUE TITLE
Variable view does not scale correctly with models with a lot of new RTD

### ISSUE THREAD

**Issue by dvojtise** [eclipse/gemoc-studio-modeldebugging#87]
The current Variable view is flat. It displays all Runtime data at the root of the view.
This doesn't scale correctly for languages with a lot of RTD or that create a lot of new RTD.

for example in the Logo (complete code in [dvojtise/mde-crashcourse-logo](https://github.com/dvojtise/mde-crashcourse-logo/tree/master/part3-mmfirst-solution))
![espace de travail 1_093](https://user-images.githubusercontent.com/661468/52283029-080b4180-2962-11e9-9323-826c32dfcba9.png)
The variable view grows by adding new segments to the drawing board. It is impossible to interpret the data displayed there especially after a high number of steps. 

I suggest that the view should display only RTD which are not already contained (directly or indirectly) by another RTD. Thus the view should display only "root" RTD and use the nesting  to display the rest.

Note1: the yellow highlighting about the last modified values should unfold the tree in order to focus on the appropriate data.
Note2: the same problem also occurs on the Multidimentionnal timeline which should probably take a similar nesting approach

**Comment by dvojtise** [comment]
Another idea to improve the view would be to add navigation to the `eContainer`. Something similar to what Modisco does. (see picture here https://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.modisco.infrastructure.doc%2Fmediawiki%2Fmodel_browser%2Fuser.html )

This would help understand the origin of the displayed elements.

**Comment by ebousse** [comment]
I agree with the problem, and the proposed solution to show data as a tree sounds reasonable to me.

The more I think of it, the more I feel like a rewrite of the legacy "dsldebugger" code that acts as an intermediate layer between GEMOC and the Eclipse debug API will be required, especially for such features.

**Comment by dvojtise** [comment]
my current status in branch https://github.com/eclipse/gemoc-studio-modeldebugging/tree/improve-variable-view
I can display the RTD either using a flat or nested representation (with a toggle button on the variable view). However, when nested, I lose the data modification highlighting (yellow) which is not applied in children nodes of the variable view (I don't know yet if I can fix that)
